client/components/common/VersionString/index.js

```diff
@@ -91,12 +90,13 @@ const VersionString = ({
 };
 
 VersionString.propTypes = {
+    date: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)])
+        .isRequired,
```

Comment:
@Paul-Clue this is the source of the failing test. Your PR seems to be causing a regression from #804 which was merged while you were working on this. [Here is the relevant changed file.](https://github.com/w3c/aria-at-app/pull/804/files#diff-5a6ef0a58a10458be0d847550d91f47e200c5f0922c30226f9e7d927d69194b5). It looks like that PR was merged since you opened this PR. I don't think you have any actual changes in this file.

Reply:
Got it. Thanks!

Reply:
Hey, @stalgiag, I just fixed issues that were causing the error. Check it out, when you have time.